### PR TITLE
Add in auth and ssl for ST configs

### PR DESCRIPTION
**Pull request by jryburn**

### CODE REVIEW

**Comment by kentik-rbarnes** [comment]
I wonder if we can also comment at the top of the file when support of some of these commands was added to Junos? Optionally we could include the reference URLs if we think it's not too cluttered?

/* The allow-commands-regexps and deny-commands-regexps statements were introduced in Junos OS Release 18.1 */

/* ssl statement introduced in Junos OS Release 16.1 */
/* mutual-authentication, client-certificate-request, and certificate-authority options added in Junos OS Release 17.4R1 */
/* hot-reloading option added in Junos OS Release 20.4R1 */
/* use-pki option added in Junos OS Evolved Release 22.2R1 */

**Comment by jryburn** [comment]
> I wonder if we can also comment at the top of the file when support of some of these commands was added to Junos? Optionally we could include the reference URLs if we think it's not too cluttered?
> 
> /* The allow-commands-regexps and deny-commands-regexps statements were introduced in Junos OS Release 18.1 */
> 
> /* ssl statement introduced in Junos OS Release 16.1 _/ /_ mutual-authentication, client-certificate-request, and certificate-authority options added in Junos OS Release 17.4R1 _/ /_ hot-reloading option added in Junos OS Release 20.4R1 _/ /_ use-pki option added in Junos OS Evolved Release 22.2R1 */

Good call. I just added this in and did another commit.

File: Juniper/MX-series/telemetry_dialin.conf [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+mutual-authentication, client-certificate-request, and certificate-authority options added in Junos OS Release 17.4R1
+hot-reloading option added in Junos OS Release 20.4R1
+use-pki option added in Junos OS Evolved Release 22.2R1
+*/
+
+system {
+    login {
+        class {
+            grpc-operator {
+                permissions all;
```
Comment (kentik-rbarnes) [inline Juniper/MX-series/telemetry_dialin.conf:16]: after some vmx testing, it looks like permissions "view" will be sufficient vs "all" to reduce new user perms.

**Review by kentik-rbarnes — APPROVED** [review]
didn't verify EOS/IOS-XR explicitly (still needing to test on a switch) but it looks good to me.

**Pull request merged**